meta-resin-common/recipes-core/images/resin-image.bb

```diff
@@ -15,7 +15,7 @@ IMAGE_ROOTFS_EXTRA_SPACE = "0"
 IMAGE_ROOTFS_MAXSIZE = "319488"
 
 # Generated resinhup-tar based on RESINHUP variable
```

Comment:
We maintain the resinhup-tar file as backwards compatible so we can leave it for now.